index.bs

```diff
@@ -1106,10 +1111,21 @@ The <dfn attribute for="XRRigidTransform">position</dfn> attribute is a 3-dimens
 
 The <dfn attribute for="XRRigidTransform">orientation</dfn> attribute is a quaternion describing the rotational component of the transform. The {{XRRigidTransform/orientation}} MUST be normalized to have a length of <code>1.0</code>.
 
-The <dfn attribute for="XRRigidTransform">matrix</dfn> attribute returns the transform described by the {{XRRigidTransform/position}} and {{XRRigidTransform/orientation}} attributes as a [=matrix=]. This attribute SHOULD be lazily evaluated.
+The <dfn attribute for="XRRigidTransform">matrix</dfn> attribute returns the transform described by the {{XRRigidTransform/position}} and {{XRRigidTransform/orientation}} attributes as a [=matrix=]. This attribute MUST be computed by [=XRRigidTransform/obtain the matrix|obtaining the matrix=] for the {{XRRigidTransform}}.
```

Comment:
@Manishearth @toji Sorry for the post-merge comment here:

The "lazy evaluated" part originally introduced in https://github.com/immersive-web/webxr/pull/560 disappeared here. As it is viable to use a DualQuaternion as view transform directly, it would make sense to keep the matrix computation optional. It's just a nit-optimization for a currently less common use case, of course.

Or does "MUST be computed" imply it's computed on access?

Reply:
Yep.

Reply:
Alright, awesome, thanks for the clarification!